### ISSUE TITLE
the /var/lib/lxcfs/cgroup/ is empty if systemd.unified_cgroup_hierarchy=1  has been set

### ISSUE THREAD

**Issue by Sn0rt** [lxc/lxcfs#505]
the  ` /var/lib/lxcfs/cgroup/` is empty when the linux `systemd.unified_cgroup_hierarchy=1`  has been set. 

the empty dir is better than a EIO message, right?  What do you guys think ? If you agree with my point of view, I can send this commit.

currently, we can find the case #313 

however , i think the follow action is better 

```bash
root@ubuntu-focal:/home/vagrant/lxcfs# cat /proc/cmdline
BOOT_IMAGE=/boot/vmlinuz-5.4.0-91-generic root=UUID=94ab4479-4a78-42cb-9888-bb63d6a79c8c ro systemd.unified_cgroup_hierarchy=1 console=tty1 console=ttyS0
root@ubuntu-focal:/home/vagrant/lxcfs# ls /var/lib/lxcfs/
cgroup  proc  sys
root@ubuntu-focal:/home/vagrant/lxcfs# ls /var/lib/lxcfs/cgroup/
```

**Comment by brauner** [comment]
Yeah, I think that makes sense.

**Comment by brauner** [comment]
Closed in #506.
